dev/src/NewComponent.php

```diff
@@ -110,7 +110,7 @@ private static function getProtoPath(string $protoFilename, ?string $version): s
         }
         $parts = explode('/', $protoPath);
         if ($i = array_search($version, $parts)) {
-            $parts[$i] = "(.*)";
+            $parts[$i] = "($version)";
```

Comment:
Not sure whether it's okay to have `$parts[$i]` like `()` when `$version` is `null`, but to retain the old behavior, you can consider:

```suggestion
            $version = $version ?? '.*';
            $parts[$i] = "($version)";
```

Reply:
You can see 3 lines above this one, if `$version` is null, the function does not reach this point. So we already know for sure that `$version` is not null.